package.json

```diff
@@ -111,7 +111,7 @@
     "@testing-library/dom": "8.11.3",
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "12.1.2",
-    "@testing-library/react-hooks": "7.0.2",
+    "@testing-library/react-hooks": "8.0.1",
```

Comment:
bumped to avoid further resolutions for react type issues ( this version doesn't specify `@types/react` within deps )